packages/server/src/app/db/content/content.service.ts

```diff
@@ -96,4 +121,25 @@ export class ContentService {
             $inc: {"stats.comments": 1}
         });
     }
+
+    /**
+     * Adds a view to some content.
+     * 
+     * @param contentId The content's ID
+     */
+    async addView(contentId: string) {
+        return await this.contentModel.updateOne({'_id': contentId}, {
+            $inc: {'stats.views': 1}
+        });
+    }
```

Comment:
This method name confused the heck out of me until I saw the implementation. I kept thinking it was adding a _UI_ view to something. Should probably rename it to `incrementViewCount()` or something.